components/properties/src/props.rs

```diff
@@ -2455,3 +2502,123 @@ impl_value_getter! {
         pub fn get_enum_to_long_name_mapper() / enum_to_long_name_mapper() -> PropertyEnumToValueNameLinearMapper / PropertyEnumToValueNameLinearMapperBorrowed;
     }
 }
+#[cfg(test)]
+mod test_enumerated_property_completeness {
+    use std::cmp::Ordering;
+
+    use super::*;
+
+    fn check_enum<'a>(
+        lookup: &PropertyValueNameToEnumMapV1<'_>,
+        consts: impl IntoIterator<Item = &'a (&'static str, u16)>,
+    ) {
+        let mut data = lookup.map.iter_copied_values().collect::<Vec<_>>();
+        data.sort_by_key(|(_, v)| *v);
+        data.dedup_by_key(|(_, v)| *v); // data may contain multiple entries for the same value
```

Comment:
Suggestion: please use `BTreeSet` or `HashSet` (this is test code). Then you can use the built-in `difference` functions and be less prone to error.